Pixel Game/Assets/Scripts/PlayerInput.cs

```diff
@@ -37,16 +37,15 @@ void Start()
 
         // TO-DO - Get the closestNPC by looping through a list of NPC to get the closest one
         closestNPC = GameObject.FindWithTag("Npc"); // This should be changed to distance calculation so we get closest npc

```

Comment:
closestNpc assignment can be removed since we are using interract script now :)

Reply:
Done